agent/installer/installer.go

```diff
@@ -49,7 +49,7 @@ func getSupportedRegistry(ob algo.OutputBuilder) registry {
 
 		// BYOH Bundle Repository. Associate bundle with installer
 		linuxDistro := "Ubuntu_20.04.1_x86-64"
-		addBundleInstaller(linuxDistro, "v1.22.1", &algo.Ubuntu20_4K8s1_22{})
+		addBundleInstaller(linuxDistro, "v1.22.3", &algo.Ubuntu20_4K8s1_22{})
```

Comment:
Better to have a global k8 version const, so that we don't need to update it everywhere in the future?

Now that we are here, I just searched and found one more place where we can update the version(`optional for this PR`).
https://github.com/vmware-tanzu/cluster-api-provider-bringyourownhost/blob/1bb756c3d190111e1d85e987eff1224e531b0bee/controllers/infrastructure/byomachine_controller_test.go#L37